### ISSUE TITLE
Json support

### ISSUE THREAD

**Issue by devzonetrack** [andrea-magni/MARS#28]
is possible to post json data?

**Comment by andrea-magni** [comment]
Hi, if you mean sending (POST) data (JSON) using the MARS Client Library, yes, it is possibile. Drop a TMARSResourceJSON and have a look at the POST method.

If you want to know how to handle data sent to the server through a POST method, let me know...

Sincerely

**Comment by devzonetrack** [comment]
Yes, i want send data to server with  TObject/Class as parameter is possible, and get response also in TObject/Class ?  i found another TRestClient component really easy to use [https://github.com/fabriciocolombo/delphi-rest-client-api](url) but unfortunately not support android platform
